main/components/__init__.py

```diff
@@ -58,15 +70,49 @@ def remove(self, id: int) -> None:
         </ul>
         {{message}}
     </div>
-    """
+    
+<script>
```

Comment:
this could be done in a `script: javascript=""" ... """` attribute, not in the `template`.